Microsoft.Azure.CosmosRepository.Samples/Paging/Program.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Bogus;
+using Microsoft.Azure.CosmosRepository;
+using Microsoft.Azure.CosmosRepository.Paging;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using static System.Console;
+
+ConfigurationBuilder configuration = new ConfigurationBuilder();
+
+ServiceProvider provider = new ServiceCollection().AddCosmosRepository(options =>
+{
+    options.CosmosConnectionString = Environment.GetEnvironmentVariable("CosmosConnectionString");
+    options.DatabaseId = "paging-db";
+    options.ContainerPerItemType = true;
+})
+.AddSingleton<IConfiguration>(configuration.Build())
+.BuildServiceProvider();
+
+IRepository<Person> repository = provider.GetRequiredService<IRepository<Person>>();
+
+await PageWithContinuationTokens();
+await JumpStraightToPage();
+// await JumpToStraightThenUseContinuationToken();
+
+
+
+
+
+
+async Task JumpToStraightThenUseContinuationToken()
+{
+    int pageNumber = 4;
+    int pageSize = 10;
+    int totalItemsRead = 0;
+    int totalItemsExpectedToRead = 60;
+    string continuationToken = null;
+
+    double totalRuCharge = 0;
+
+    //TODO: this currently fails on the second pass when using the continuation token with a 400 bad request for a malformed token not entirely sure why yet.
```

Comment:
Having some trouble with this example the initial jump the page works correctly but then the subsequent request using a continuation token fails with a 400 bad request saying the token is malformed I will get a sample of the token and paste it here. 

I did look at this alternate implementation which did work but would hog memory & gets slower per page being iterated. I did not think this was practical https://gist.github.com/mumby0168/5553fd11dfac0cd4cd42c5a53ef121ae

Reply:
This is the exception message

```
Unhandled exception. Microsoft.Azure.Cosmos.CosmosException : Response status code does not indicate success: BadRequest (400); Substatus: 0; ActivityId: ; Reason: (Response status code does not indicate success: BadRequest (400); Substatus: 0; ActivityId: ; Reason: (Response status code does not indicate su
ccess: BadRequest (400); Substatus: 0; ActivityId: ; Reason: (Response status code does not indicate success: BadRequest (400); Substatus: 0; ActivityId: ; Reason: (Invalid format for continuation token {"limit":2147483637,"sourceToken":"{\"offset\":0,\"sourceToken\":\"[{\\\"token\\\":\\\"+RID:~pR5MANaj6wwoA
AAAAAAAAA==#RT:4#TRC:40#ISV:2#IEO:65567#QCF:7#FPC:ASgAAAAAAAAAZAAAAAAAAAA=\\\",\\\"range\\\":{\\\"min\\\":\\\"\\\",\\\"max\\\":\\\"FF\\\"}}]\"}"} for ParallelCrossPartitionQueryPipelineStage););););
 ---> Microsoft.Azure.Cosmos.Query.Core.Exceptions.MalformedContinuationTokenException: Invalid format for continuation token {"limit":2147483637,"sourceToken":"{\"offset\":0,\"sourceToken\":\"[{\\\"token\\\":\\\"+RID:~pR5MANaj6wwoAAAAAAAAAA==#RT:4#TRC:40#ISV:2#IEO:65567#QCF:7#FPC:ASgAAAAAAAAAZAAAAAAAAAA=\\\
",\\\"range\\\":{\\\"min\\\":\\\"\\\",\\\"max\\\":\\\"FF\\\"}}]\"}"} for ParallelCrossPartitionQueryPipelineStage


```